### PR TITLE
add ProfilePage, add Button, add classNames

### PR DESCRIPTION
**Pull request by EduardDE7**

### CODE REVIEW

File: src/pages/ProfilePage.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  CreditCardIcon,
+  PrivacyIcon,
+} from '../components/ui/icons';
+
+const ProfilePage = () => {
+  return (
+    <div className="flex pt-[50px] pb-[35px] flex-col items-center mx-4 font-normal">
+      <div className="w-[100px] h-[100px] mb-[20px]">
+        <img src="/src/assets/images/profile.png" alt="" />
+      </div>
```
Comment (OsmanAlb) [inline src/pages/ProfilePage.jsx:17]: To add change image.)

File: src/pages/ProfilePage.jsx
Comment (OsmanAlb) [inline src/pages/ProfilePage.jsx:1]: What do you think

**Review by OsmanAlb — CHANGES_REQUESTED** [review]
Change the name

**Comment by EduardDE7** [comment]
will be done

**Review by OsmanAlb — COMMENTED** [review]
all good

**Pull request merged**